cmd/manager-hub/main.go

```diff
@@ -69,9 +71,9 @@ func main() {
 	logConfig := textlogger.NewConfig()
 	logConfig.AddFlags(flag.CommandLine)
 
-	var configFile string
+	var userConfigName string
 
-	flag.StringVar(&configFile, "config", "", "The path to the configuration file.")
```

Comment:
I think we can keep the old names. A config file can be applied by users as well. No need to call it user's config file IMO. This will also narrow down this PR to the bare minimum required to solve the issue.

Reply:
Update: Let's call it `userConfigMapName`?